doc/source/tutorial/tutorial_transforming.ipyml

```diff
@@ -30,66 +28,117 @@ cells:
     from larray import __version__
     __version__
 
-  id: 2
 
-- markdown: |
-    ## Manipulating axes
+- code: |
+    # load the 'demography_eurostat' dataset
+    demo_eurostat = load_example_data('demography_eurostat')
 
 
 - code: |
-    # let's start with
-    pop = load_example_data('demography').pop[2016, 'BruCap', 90:95]
+    # get a copy of the 'pop' array from the 'demography_eurostat' dataset
+    pop = demo_eurostat.pop.copy()
     pop
 
-  id: 3
+
+- markdown: |
+    ## Manipulating axes
+    
+    
+    LArray offers several to manipulate the axes and labels of an array:
+    - [set_labels](../_generated/larray.LArray.set_labels.rst#larray.LArray.set_labels): to replace all of some labels of one or several axes.
+    - [rename](../_generated/larray.LArray.rename.rst#larray.LArray.rename): to replace one of several axis names.
+    - [set_axes](../_generated/larray.LArray.set_axes.rst#larray.LArray.set_axes): to replace one of several axes.
+    - [transpose](../_generated/larray.LArray.transpose.rst#larray.LArray.transpose): to modify the order of axes.
+    - [drop](../_generated/larray.LArray.drop.rst#larray.LArray.drop): to remove one or several labels.
+    - [combine_axes](../_generated/larray.LArray.combine_axes.rst#larray.LArray.combine_axes): to combine axes.
+    - [split_axes](../_generated/larray.LArray.split_axes.rst#larray.LArray.split_axes): to split one or several axes.
```

Comment:
add something like "by splitting their labels", otherwise we could think it is splitting axes in chunks